ee/clickhouse/models/test/test_property.py

```diff
@@ -442,6 +446,38 @@ def test_parse_prop_clauses_defaults(snapshot):
     assert parse_prop_clauses(filter.properties, None, person_properties_mode=PersonPropertiesMode.EXCLUDE) == snapshot
 
 
+TEST_BREAKDOWN_PROCESSING = [
+    ("$browser", "events", "properties", "prop", "trim(BOTH '\"' FROM JSONExtractRaw(properties, '$browser')) AS prop"),
+    (
+        ["$browser"],
+        "events",
+        "properties",
+        "value",
+        "array(trim(BOTH '\"' FROM JSONExtractRaw(properties, '$browser'))) AS value",
+    ),
+    (
+        ["$browser", "$browser_version"],
+        "events",
+        "properties",
+        "prop",
+        "array(trim(BOTH '\"' FROM JSONExtractRaw(properties, '$browser')),trim(BOTH '\"' FROM JSONExtractRaw(properties, '$browser_version'))) AS prop",
```

Comment:
I have no idea if its relevant, but there are some places in the code where we choose to pull from a "materialized" column, I'm not sure the exact criterion for this is but I expect that we'll need to generate exprs for multi breakdowns. Possible `get_property_string_expr` handles that condition, but maybe useful to include a test example of this as well.

Reply:
100%! The criteria is whether the property is materialised or not, and yes, `get_property_string_expr` handles that!

To add a test for this, use the `@test_with_materialized_columns` decorator